### PR TITLE
Add GET /groups/me route

### PR DESCRIPTION
**Pull request by woodwardmw**

### CODE REVIEW

File: security_routes/admin_routes.py
Comment (alejandroaquintero) [inline security_routes/admin_routes.py:99]: looks pretty simple but i think we should a test in the admin flow of test admin routes, it also serves as documentation

**Review by alejandroaquintero — APPROVED** [review]

File: security_routes/admin_routes.py
Comment (woodwardmw) [reply security_routes/admin_routes.py:99]: OK yep. I've now added a test. I'm a bit scared to run it before I've merged your updates! So I think I'll merge this, and if it fails the test I'll fix it.

**Pull request merged**
